src/main/java/com/commercetools/helper/mapper/PaymentMapper.java

```diff
@@ -0,0 +1,21 @@
+package com.commercetools.helper.mapper;
+
+import com.commercetools.model.CtpPaymentWithCart;
+import com.paypal.api.payments.Payment;
+
+import javax.annotation.Nullable;
+
+public interface PaymentMapper {
+
+    /**
+     * See Chapter <b><code>04. Payments</code></b> of <i>Integrating PayPal PLUS</i> guide.
+     *
+     * @param ctpPaymentWithCart commercetools cart and payment to map
+     * @return Paypal Plus {@link Payment} with respective values from {@link CtpPaymentWithCart}
+     * @see <a href="https://www.paypalobjects.com/webstatic/downloads/PayPal-PLUS-IntegrationGuide.pdf">
```

Comment:
From this doc and chapter 4, it's not very clear how to set up the payment. Did you use any other resources as well?